### ISSUE TITLE
 warning: ignoring return value of ‘system’ declared with attribute ‘warn_unused_result’ [-Wunused-result]  2333 |   if(arg->i == 0)  system("killall bar.sh");

### ISSUE THREAD

**Issue by ajrances** [siduck/chadwm#80]

**Comment by siduck** [comment]
does everything work?

**Comment by ajrances** [comment]
yes but whenever i change the config.h to use terminal st as my command line it does not save

**Comment by siduck** [comment]
im unable to produce this error :l , could you show the terminal output?

**Comment by ajrances** [comment]
i have installed dwm before should i remove all of it?

**Comment by ajrances** [comment]
i hope it does not interfere with ur script

**Comment by siduck** [comment]
> i have installed dwm before should i remove all of it?

when was the last time u updated chadwm? like a git pull

**Comment by ajrances** [comment]
![wget-log 3](https://user-images.githubusercontent.com/96360675/158537601-f17557c2-01fe-48a8-bfca-77c1febeef1a.png)

**Comment by ajrances** [comment]
> > i have installed dwm before should i remove all of it?
> 
> when was the last time u updated chadwm? like a git pull

i dont update it i just installed it using ur guide on github

**Comment by siduck** [comment]
> > > i have installed dwm before should i remove all of it?
> > 
> > 
> > when was the last time u updated chadwm? like a git pull
> 
> i dont update it i just installed it using ur guide on github

yes, when did u did that?

**Comment by ajrances** [comment]
> 

just recently today

**Comment by siduck** [comment]
![image](https://user-images.githubusercontent.com/59060246/165895886-73a0975c-8d8c-40b5-9e68-1f6c2e924a55.png)

this should be the output

**Comment by siduck** [comment]
@ajrances clone the repo again and compile it
